pkg/webservice/dao/cluster_info.go

```diff
@@ -18,18 +18,22 @@
 package dao
 
 type ClusterDAOInfo struct {
-	PartitionName         string `json:"partition"`
-	ClusterName           string `json:"clusterName"`
-	TotalApplications     string `json:"totalApplications"`
-	FailedApplications    string `json:"failedApplications"`
-	PendingApplications   string `json:"pendingApplications"`
-	RunningApplications   string `json:"runningApplications"`
-	CompletedApplications string `json:"completedApplications"`
-	TotalContainers       string `json:"totalContainers"`
-	FailedContainers      string `json:"failedContainers"`
-	PendingContainers     string `json:"pendingContainers"`
-	RunningContainers     string `json:"runningContainers"`
-	ActiveNodes           string `json:"activeNodes"`
-	TotalNodes            string `json:"totalNodes"`
-	FailedNodes           string `json:"failedNodes"`
+	ScheduleStartDate     string              `json:"scheduleStartDate"`
+	RMBuildInformation    []map[string]string `json:"rmBuildInformation"`
+	PartitionName         string              `json:"partition"`
+	ClusterName           string              `json:"clusterName"`
+	TotalApplications     string              `json:"totalApplications"`
+	FailedApplications    string              `json:"failedApplications"`
+	PendingApplications   string              `json:"pendingApplications"`
+	RunningApplications   string              `json:"runningApplications"`
+	CompletedApplications string              `json:"completedApplications"`
+	TotalContainers       string              `json:"totalContainers"`
+	FailedContainers      string              `json:"failedContainers"`
+	PendingContainers     string              `json:"pendingContainers"`
+	RunningContainers     string              `json:"runningContainers"`
+	ActiveNodes           string              `json:"activeNodes"`
+	TotalNodes            string              `json:"totalNodes"`
+	FailedNodes           string              `json:"failedNodes"`
 }
+
+var ScheduleStartDate string
```

Comment:
Move this as a variable of type _time.Time_ with the name startTime ​into the `ClusterContext` object.
Set it as part of the `NewClusterContext` and `newClusterContext` calls.
Add a locked method to the clustercontext `GetStartTime() time.Time` to retrieve the value. 

That should also remove the dao dependency from the entrypoint which is much cleaner